dev/test_data.sql

```diff
@@ -189,36 +191,36 @@ CREATE OR REPLACE VIEW inventory.hosts AS
 
 GRANT SELECT ON TABLE inventory.hosts TO cyndi_reader;
 
-INSERT INTO inventory.hosts_v1_0 (id, account, display_name, tags, updated, created, stale_timestamp, system_profile) VALUES
-('00000000000000000000000000000001', '1', '00000000-0000-0000-0000-000000000001', '[{"key": "k1", "value": "val1", "namespace": "ns1"},{"key": "k2", "value": "val2", "namespace": "ns1"}]',
+INSERT INTO inventory.hosts_v1_0 (id, insights_id, account, display_name, tags, updated, created, stale_timestamp, system_profile) VALUES
+('00000000000000000000000000000001', '00000000-0000-0000-1111-000000000001', '1', '00000000-0000-0000-0000-000000000001', '[{"key": "k1", "value": "val1", "namespace": "ns1"},{"key": "k2", "value": "val2", "namespace": "ns1"}]',
```

Comment:
I'm a bit obsessed, I know. But could you please make testing insights_id values `00000000-0000-0000-0001-000000000000` - `00000000-0000-0000-0011-000000000000`. It would be nicer. Thanks very much.